config/clusters/2i2c-uk/lis.values.yaml

```
@@ -63,7 +63,10 @@ jupyterhub:
           mem_limit: 2G
           mem_guarantee: 512M
       - display_name: Large Instance
-        description: "Provides 1G - 4G of RAM"
+        # Requested in https://2i2c.freshdesk.com/a/tickets/1348
+        description: |
```

I'm not sure, it didn't seem to work by adding a </br> inside the <p> tag. I figure maybe this will lead to kubespawner to use something else than <p> to wrap the description text? I'm not sure - for now, lets inline the comment about this I figure.

```suggestion
        # FIXME: The </p><p> is a hack for kubespawner to render a new line, maybe there are better ways to do this?
        description: |
```